tests/integration/test_storage_kafka/test.py

```diff
@@ -233,6 +233,81 @@ def test_kafka_settings_new_syntax(kafka_cluster):
     members = describe_consumer_group('new')
     assert members[0]['client_id'] == u'instance test 1234'
 
+
```

Comment:
Please, at least provide links to both issues or, even better, put some issue description here.